pdns/dnsdist-lua-actions.cc

```diff
@@ -1173,16 +1173,16 @@ class KeyValueStoreLookupAction : public DNSAction
 };
 
 template<typename T, typename ActionT>
-static void addAction(GlobalStateHolder<vector<T> > *someRulActions, luadnsrule_t var, std::shared_ptr<ActionT> action, boost::optional<luaruleparams_t> params) {
+static void addAction(GlobalStateHolder<vector<T> > *someRulActions, luadnsrule_t& var, std::shared_ptr<ActionT>& action, boost::optional<luaruleparams_t>& params) {
```

Comment:
I don't know how strict we want to be with respect to const-correctness, but you can probably add some `const` for these parameters.

Reply:
Thanks, fixed!